docs/offline.rst

```diff
@@ -15,7 +15,14 @@ Rally will automatically detect upon startup that no Internet connection is avai
 
     [WARNING] No Internet connection detected. Automatic download of track data sets etc. is disabled.
 
-It detects this by trying to connect to ``github.com``. If you want to disable this probing you can explicitly specify ``--offline``.
+It detects this by trying to connect to ``https://github.com``. If you want it to probe against a different HTTP endpoint (e.g. a company-internal git server) you need to add a configuration property named ``probing.url`` in the ``system`` section of Rally's configuration file at ``~/.rally/rally.ini``. Specify ``--offline`` if you want to disable probing entirely.
+
+Example of `system` section with custom probing url in ``~/.rally/rally.ini``::
+
+    [system]
+    env.name = local
+    probing.url = "https://www.company-internal-server.com/"
```

Comment:
The double-quotes are not required and we also don't use them elsewhere in Rally. Can you please remove them?

Reply:
Done!